src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/opentelemetry/proto/resource/v1/resource.proto

```diff
@@ -22,6 +22,6 @@ option java_outer_classname = "ResourceProto";
 
 // Resource information.
 message Resource {
-  // Set of labels that describe the resource.
-  map<string,string> labels = 1;
+  // Set of attributes that describe the resource.
+  map<string,string> attributes = 1;
```

Comment:
I believe these proto files are out of date and needs to be changed to accommodate OTLP changes (which are in progress). Changing the name here is not required now.

Reply:
OK. Would you rather leave it unchanged then instead of at least aligning with the API change (naming) this PR does?
If we're moving on with this PR I can rollback this change.

Reply:
I'd suggest to leave them unchanged and then update all of them in a single commit. So schemas are consistent

Reply:
Will do